tap_jira/streams.py

```diff
@@ -84,6 +96,23 @@ def sync_sub_streams(page, issue_changelog_updated):
                     for changelog in page:
                         changelogs_to_write.append(changelog)
 
+
+            for changelog in changelogs_to_write:
+                changelog_items = []
+                for item in changelog['items']:
+                    if 'fieldId' in item and should_exclude_field(item['fieldId'], item['field']):
+                        if item['from'] is not None:
```

Comment:
I think we will also want to pass through empty strings since those show up in the from and to fields it looks like.